client/components/place/place-list/place-list.component.js

```diff
@@ -8,10 +8,14 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         var ctrl = this;
         var places = [];
         var tracks = [];
-        var placeCounter;
+        var placeCounter = 1;
         var trackCounter;
         var placeTypeLength;
         var trackTypeLength;
+        var key;
+        var map;
+
+        var realPlacesType = [];
```

Comment:
places - is an array of place objects {id, name, location, etc.}
realPlacesType - is only types of active places, which are on map `['camp', 'service']` for good search in DB